_posts/2024-02-27-comet-donation.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Announcing Apache Arrow DataFusion Comet"
+date: "2024-02-27 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+The Apache Arrow PMC is pleased to announce the donation of the [Comet project],
+a native Spark Accelerator built on [Apache Arrow DataFusion].
+
+Comet is a Apache Spark Accelerator that uses Apache Arrow DataFusion to
+accelerate Spark workloads. It is written in Rust and designed as a drop-in
+replacement for Spark's JVM based execution engine and offers significant
+performance improvements for some workloads as shown below.
+
+```text
+┌───────────────────────────┐
+│           Spark           │
+│   Frontend / Optimizer    │
+│        (JVM Based)        │
+└───────────────────────────┘
+              │              
+              ▼              
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃           Comet           ┃
+┃     Execution Engine      ┃
+┃       (Native Code)       ┃
+┗━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+```
+
```

Comment:
I felt we needed some sort of image, but this is pretty basic ASCII art... If anyone has other ideas that would be appreciated

Reply:
The image in [comet's README](https://github.com/apache/arrow-datafusion-comet/raw/main/doc/comet-overview.png) is a good alternative, but we need to add the driver part.

Reply:
That is a good image, but it is quite detailed

What I was hoping was to have an image that visually depicts the relationship between Spark and Comet 

Something similar maybe to https://github.com/oap-project/gluten?tab=readme-ov-file#2-architecture

![image](https://github.com/apache/arrow-site/assets/490673/2327a5f2-45a0-4879-a5a0-1e61c40e58b9)

I don't know enough about how spark is implemented internally to make such a picture now, though

Reply:
I'm thinking a figure like:

Spark Frontend / Optimizer (JVM Based) -> Comet Execution Engine (Native Code)

* -> Arrow DataFusion
* -> Spark compatible expression/operator

Reply:
I took a shot at updating it in 70ff88d

What do you think?

<img width="856" alt="Screenshot 2024-02-28 at 6 50 30 AM" src="https://github.com/apache/arrow-site/assets/490673/aa46f771-d1c4-4437-91fb-c09829e22a09">